### ISSUE TITLE
Can I know the UUID of my own device?

### ISSUE THREAD

**Issue by geert56** [Pushjet/Pushjet-Server-Api#28]
Not really an issue. Nice piece of work!
Took a while to get the concept under control, but now I am happy
to create services, subscribe and send messages.
Would it be possible to learn the device UUID that is used in the Android app?
There is no API function to get the device subscribers to a service.
Knowing my device uuid I would be able to read messages on another device
(e.g. laptop via curl or npm pushjet or python).

All the best.
Are you Dutch Bas? I studied and worked at TUE.

**Comment by Mechazawa** [comment]
yes I'm Dutch and also from Eindhoven.  I'm locally doing a rewrite of both the api and the app.  I'm not gonna add a feature to get the list of subscribers due to the fact that getting a list of uuids gives you the ability to make them subscribe to other unwanted services. I can add the subscriber count if you want and some other info. I'll add a feature to get the device uuid soon on Android when I have some spare time.

**Comment by geert56** [comment]
Thanks, that's all I wanted: to be able to see the UUID assigned to the Android app.
Good luck with your rewrite work.
